RELEASE.md

```diff
@@ -1,16 +1,36 @@
-Release
-=======
+# Release Process
 
-Although not very tricky, it is quite easy to deploy something that
-doesn't quite work as expected. The following steps navigate a through
-some of the release gotchas and will hopefully result in a successful
-release.
+Although not very tricky, it is quite easy to deploy something that doesn't
+quite work as expected. The following steps help you navigate through some of
+the release gotchas and will hopefully result in a successful release.
 
-STEPS:
-------
+---
+
+## Preparation
+
+### Communication
+
+- Ensure that homu isn't presently processing any PRs.
+- Post a note in [#dev-ember-cli](https://embercommunity.slack.com/archives/dev-ember-cli) letting us know you're doing a release.
+
+> I'm starting an Ember CLI release. Please hold off merging PRs, "homu r+"-ing, and pushing new code!
+
+### Environment
+
+Make sure that you're running the most recent stable `node` and bundled `npm`.
+
+```sh
+$ node --version
+$ npm --version
+$ cd ~/repos/ember-cli
+$ git clean -fdx
```

Comment:
this step is already at https://github.com/ember-cli/ember-cli/blob/master/dev/prepare-release#L3